profiles/common/modules/features/nexteuropa_trackedchanges/NETCInfoController.inc

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * @file
+ * Contains NextEuropaTrackedChangesInfoController.
+ */
+
+/**
+ * Class NETCInfoController.
+ */
+class NETCInfoController extends EntityAPIController {
+
+  /**
+   * Override the save method.
+   *
+   * It sets the Scan date and the label (if not set) before saving.
+   */
+  public function save($entity, DatabaseTransaction $transaction = NULL) {
+    if (empty($entity->rel_entity_type)) {
+      throw new Exception('NextEuropaTrackedChangesInfo entity_type missing');
+    }
+
+    if (empty($entity->rel_entity_id)) {
+      throw new EntityMalformedException('NextEuropaTrackedChangesInfo entity_id missing');
+    }
+
+    // Forced data setting.
+    $entity->scanned = REQUEST_TIME;
```

Comment:
I check and that goes against the lowerCamal rule of PHPCS if I declare it in the NETCInfo object, if I do that for all properties like `rel_entity_label` but I will use the entity metadat wrapper instead and see.